instrumentation/khttp/src/test/groovy/com/splunk/opentelemetry/KHttpClientTest.groovy

```diff
@@ -0,0 +1,48 @@
+package com.splunk.opentelemetry
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+import io.opentelemetry.instrumentation.test.AgentTestTrait
+import io.opentelemetry.instrumentation.test.base.HttpClientTest
+import khttp.KHttp
+
+class KHttpClientTest extends HttpClientTest<Void> implements AgentTestTrait {
+
+    @Override
```

Comment:
Shouldn't this file be formatted with a 2-space indent?

Reply:
Yeah, it seems spotless does not apply to groovy here, as we haven't any before